specification/security/resource-manager/Microsoft.Security/preview/2021-01-01-preview/examples/secureScoreControlDefinitions/ListSecureScoreControlDefinitions_subscription_example.json

```diff
@@ -0,0 +1,83 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01-preview",
+    "scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Security/SecureScoreControlDefinitions/a9909064-42b4-4d34-8143-275477afe18b",
```

Comment:
The ids in this file should start with the subscription scope

Reply:
Currently, there are only tenant level definitions, that should be listed in all the tenant's resources.
There are no subscription's specific definitions